### ISSUE TITLE
FBA: edits to transitive directives do not trigger project reload

### ISSUE THREAD

**Issue by RikkiGibson** [dotnet/roslyn#85112]
Scenario:

```cs
// App.cs
#!/usr/bin/env dotnet
#:include Util.cs
```

```cs
// Util.cs
new Util2();
```

```cs
// Util2.cs
public class Util2 { }
```

Steps:

1) Write above files with indicated contents to some directory.
2) Open directory in vscode.
3) Observe semantic error in `Util.cs`
3) Insert `#:include Util2.cs` into `Util.cs` and save.

**Expected:** Semantic error in `Util.cs` disappears.
**Actual:** Semantic error remains. Editing `App.cs` will make the error in `Util.cs` disappear.

This issue is referenced in source
